test/e2e/mcd_test.go

```diff
@@ -40,3 +43,59 @@ func TestMCDToken(t *testing.T) {
 		}
 	}
 }
+
+func TestSSHAccessedAnnotation(t *testing.T) {
+	cb, err := common.NewClientBuilder("")
+	if err != nil {
+		t.Errorf("%#v", err)
+	}
+	k := cb.KubeClientOrDie("test-ssh-accessed")
+
+	nodes, err := k.CoreV1().Nodes().List(metav1.ListOptions{})
+	if err != nil {
+		t.Errorf("failed to list nodes %v", err)
+	}
+
+	var (
+		nodeName string
+		nodeIP   string
+	)
+	for _, node := range nodes.Items {
+		for _, addr := range node.Status.Addresses {
+			// just pick a master with an external IP
+			if addr.Type == v1.NodeExternalIP {
+				nodeIP = addr.Address
+				nodeName = node.Name
+			}
+		}
+	}
+
+	node, err := k.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
+	if err != nil {
+		t.Errorf("cannot get node %q: %v", nodeName, err)
+	}
+	sshAnnotation, ok := node.ObjectMeta.Annotations["machineconfiguration.openshift.io/ssh"]
```

Comment:
nit: It may be worth using the constants: https://github.com/openshift/machine-config-operator/blob/master/pkg/daemon/constants.go#L34-L37